README.md

```diff
@@ -0,0 +1,34 @@
+# Microverse Module 1 Capstone Project
+
+> This project is a website about live events booking site I used my own content and the designed is inspired by Cindy Shin on Behance.
+
+## Built with
+
+- HTML, CSS and Javascript
+- Fontawesome icons
+
+## Live Demo
+
+[Live Demo Link]
+
+## Getting started
+
+You can clone the repo or just follow the live link.
+
+### Prerequites
+
+- Any browser with thw internet.
+
+### Authors
+
+- Github: [@SaneMyburg](https://github.com/@SaneMyburg)
+- Twitter: [@SaneMyburg](https://twitter.com/@SaneMyburg)
+- LinkedIn: [@SaneMyburg](https://linkedin.com/@SaneMyburg)
+
+## Contibutions
+
+All contibutions, issues and features are welcome
+
```

Comment:
- [ ] Please add an acknowledgement section that gives appropriate credit to [Cindy Shin](https://www.behance.net/adagio07) the [author of the original design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), as required by the [Creative Commons License](https://creativecommons.org/licenses/).that gives appropriate credit to [Cindy Shin](https://www.behance.net/adagio07) the [author of the original design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), as required by the [Creative Commons License](https://creativecommons.org/licenses/). 👌